src/main/java/at/hannibal2/skyhanni/features/gui/MovableHotBar.kt

```diff
@@ -0,0 +1,38 @@
+package at.hannibal2.skyhanni.features.gui
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.GuiEditManager
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.transform
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraftforge.client.event.RenderGameOverlayEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class MovableHotBar {
+
+    private val config get() = SkyHanniMod.feature.gui.hotbar
+
+    private var post = false
+
+    fun isEnabled(): Boolean = config.editable && (config.showOutsideSkyblock || LorenzUtils.inSkyBlock)
```

Comment:
I feel generally in the codebase this isEnabled function is at the bottom of the file